<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modificar_juego</class>
 <widget class="QDialog" name="modificar_juego">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modificar Juego</string>
  </property>
  <widget class="QComboBox" name="cb_mj_juegos">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_mj_clasificacion">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Ninos</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Toda Persona</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Todos 10+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Adolecente</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Maduro 17+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Solo Adultos</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>En Espera</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Clasificacion:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_mj_nombre">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Precio:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Modificar</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_mj_genero">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Accion</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Disparos</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Estrategia</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Simulacion</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Deporte</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Carreras</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Aventura</string>
    </property>
   </item>
  </widget>
  <widget class="QDoubleSpinBox" name="dsb_mj_precio">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Genero:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
